saleor/graphql/core/mutations.py

```diff
@@ -357,12 +346,19 @@ def __init_subclass_with_meta__(
             return_field_name = get_model_name(model)
         if arguments is None:
             arguments = {}
-        fields = get_output_fields(model, return_field_name)
 
         _meta.model = model
         _meta.return_field_name = return_field_name
         _meta.exclude = exclude
         super().__init_subclass_with_meta__(_meta=_meta, **options)
+
+        model_type = cls.get_type_for_model()
+        if not model_type:
+            raise ImproperlyConfigured(
+                "Unable to find type for model %s in graphene registry" % model.__name__
+            )
+        fields = {return_field_name: graphene.Field(model_type)}
```

Comment:
Nitpick but would be nice if the metaclass supported adding a description to the return field.

Reply:
@patrys  Can we create a separate task for this(It's a lot of descriptions to add)? 

Should we do it by adding `output_field_description` to `ModelMutationOptions`?

Reply:
@fowczarek Create a new issue for that.

Reply:
Created: https://github.com/mirumee/saleor/issues/5807 @fowczarek Please check if everything is correct